server.js

```diff
@@ -3492,27 +3492,49 @@ function mapNugetFeed(pattern, offset, getInfo) {
   var vPreRegex = new RegExp('^\\/' + pattern + '\\/vpre\\/(.*)\\.(svg|png|gif|jpg|json)$');
 
   function getNugetVersion(apiUrl, id, includePre, request, done) {
-    var reqUrl = apiUrl + '/flatcontainer/' + id.toLowerCase() + '/index.json';
-    request(reqUrl, function(err, res, buffer) {
+	// get service index document
+	var serviceIndexUrl = apiUrl + '/index.json';	
+	request(serviceIndexUrl, function(err, res, buffer) {
```

Comment:
Having two successive requests will make this badge twice as slow to load. This request is superfluous.

Please directly call `https://api-v2v3search-0.nuget.org/autocomplete` (you are very welcome to alternate `-0-` and `-1-` to help their server load).

(Unless your assessment is that this endpoint changes faster than I can review patches, that is, about twice a year. In that case, please perform the request only once, and cache the result. For that, you can use the `regularUpdate()` function defined in server.js, with an interval corresponding to your estimated frequency of updates to the endpoint path.)